recipes/nsync/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "nsync" %}
+{% set version = "1.20.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 692f9b30e219f71a6371b98edd39cef3cbda35ac3abc4cd99ce19db430a5591a
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.0
+
+test:
+  commands:
```

Comment:
Can you add a test for the Windows build to ensure empty packages aren't uploaded?

Reply:
```suggestion
  commands:
    - if not exist ${PREFIX}/lib/libnsync.a exit 1  # [win]
    - if not exist ${PREFIX}/lib/libnsync_cpp.a exit 1  # [win]
```

Reply:
> Can you add a test for the Windows build to ensure empty packages aren't uploaded?

have fixed them